src/lib/utils2.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Converts an input string into a string array of selected values.
+ *
+ * The input string may be the value of a multi-selection template variable. In this case the input string is
+ * delimited by curly braces that enclose a comma separated list of values.
+ *
+ * The dropUnresolved parameter determines if an empty array is returned if the input starts with '$'.
+ * The dropAll parameter determines if an empty array is returned if the input is "all".
+ */
+export function processSelectionVariable(input: string, dropUnresolved: boolean, dropAll: boolean): string[] {
```

Comment:
Can the code in `utils2.ts` be merged into `utils` now? I presume it was a separate file just so you could write it in typescript.

Reply:
Yes, they can be merged.